packages/app-earn-ui/src/helpers/get-earnings-estimations-map.ts

```
@@ -0,0 +1,84 @@
+  }
+
+  const getPointInTime = (time: Dayjs) => {
+    const point = forecastData.dataPoints.daily.find((p) => {
+      return (
+        dayjs(p.timestamp).startOf('day').isSame(time, 'day') ||
+        dayjs(p.timestamp).startOf('day').subtract(1, 'day').isSame(time, 'day') // 3y needs -1 day
+      )
+    })
+
+    if (!point) {
+      console.log(
```

ESLint

[error] Unexpected console statement.

(no-console)